nilearn/reporting/data/html/report_body_template_niftimapsmasker.html

```diff
@@ -8,20 +8,27 @@
     <div class="image">
       <!-- prettier-ignore -->
       <script>
-        {{js_content}}
+        { { js_content } }
       </script>
-      <div class="btn-container">
-        <button id="prev-btn-{{unique_id}}" class="btn">Previous Map</button>
-      </div>
 
-      <div class="btn-container">
-        <h3 class="comp-title">
-          Component <span id="comp-{{unique_id}}"></span>
+      <div
```

Comment:
after the change

![image](https://github.com/user-attachments/assets/f0785cda-3fdf-4bb5-943c-acdc5bd59a6e)

https://output.circle-artifacts.com/output/job/cbdfdb12-85df-4156-b28b-64531200a796/artifacts/0/dev/auto_examples/03_connectivity/plot_probabilistic_atlas_extraction.html#sphx-glr-auto-examples-03-connectivity-plot-probabilistic-atlas-extraction-py

Reply:
The buttons don't change the maps for some reason on the link above. Is that expected?

The one on dev docs works: https://nilearn.github.io/dev/auto_examples/03_connectivity/plot_probabilistic_atlas_extraction.html#extract-the-time-series

Reply:
Looks like the javascript isn't inserted properly?

<img width="483" alt="Screenshot 2025-03-11 at 4 43 51 PM" src="https://github.com/user-attachments/assets/2809e53f-7262-4e5b-b015-9f79b4e1b598" />

Reply:
good catch!!!